frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md

```diff
@@ -30,14 +32,33 @@
   5. [k8sPhase](#5-k8sPhase)
   6. [k8sPhaseReason](#6-k8sPhaseReason)
   7. [k8sResourcePrefix](#7-k8sResourcePrefix)
-
+---------------------------------------
 
 #### [DEPRECATED Descriptors](#deprecated-descriptors-1)
+  15. [fieldGroup](#15-fieldgroup)
+  16. [arrayFieldGroup](#16-arrayFieldGroup)
+
 
 ## specDescriptors
-A specDescriptor is for describing the property of a `spec` field in your Custom Resource. Each specDescriptor provides UI widget(s) for creating/mutating CR instance on different views (i.e. CREATION, DISPLAY, and MODIFY views) in the OpenShift Console.
+A specDescriptor is for describing the property of a `spec` field in your Custom Resource. Each entry should contain the following:
```

Comment:
This information is applicable to both specDescriptor and statusDescriptor. Should the general explanation of the common fields be in it's own section? Also, the readme file already provides this info, so maybe we could just link to the appropriate section of that doc for the general info about descriptors?

Reply:
cool, thanks for pointing this out. Let's go for a new "Getting Started" section as a follow-up as you recommended.